loop.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+update_status() {
+  /app/status.sh > /app/public/status.json 2>&1
+  echo "Status updated!"
+}
+
+update_incidents() {
+  /app/incidents.sh > /app/public/incidents.json 2>&1
+  echo "Incidents updated!"
+}
+
+while true; do
+  update_status
+  update_incidents
```

Comment:
Call update incidents during the Docker build. The incidents csv files can't be update at runtime

Reply:
@ryan-bennett Mentioned yesterday there will be another PR doing all those changes